tests/Drupal/Commenting/ClassCommentUnitTest.1.inc

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ * Testing class/trait comments.
```

Comment:
any reason we need to start a new file?

We could just expand ClassCommentUnitTest.inc?

Reply:
The ClassCommentUnitTest.inc currently tests the "Missing" part of the ClassComment rule. We could merge place this test in this file but it would replace the existing test, as it can't be missing and too short at the same time.

Reply:
I meant adding a second class declaration to the test file, like in Drupal 7 code. Anyway, I don't care too much - having a second test file is also fine.

Reply:
Ah, yes that makes sense, I'll merge the test into the existing test file and then merge this. Thanks for the review!